actions/retweet.py

```diff
@@ -0,0 +1,70 @@
+from twitter import Twitter
+from twitter import OAuth
+
+import requests
+import os
+from io import StringIO
+
+from st2common.runners.base_action import Action
+
+# 1 MB chunks
+DOWNLOAD_CHUNK_SIZE_BYTES = 1 * 1024 * 1024
+
+__all__ = [
+    'UpdateStatusAction'
+]
+
+
+class UpdateStatusAction(Action):
```

Comment:
Guessing `UpdateStatusAction` is a copy/paste thing and should be `RetweetAction` ?